File: client/java/src/avt/MoneyPut.java
package avt;

import javax.microedition.lcdui.Graphics;
import main.Canvas;

public final class MoneyPut {
   private int x;
   private int y;
   int valuea;
   private int typePaint;

   public MoneyPut(int var1, int var2, int var3, int var4) {
      this.x = var1;
      this.y = var2;
      this.valuea = var3;
      this.typePaint = var4;
   }

   public final void paint(Graphics var1) {
      ImageIcon var2;
      if ((var2 = AvatarData.getImgIcon((short)(Canvas.w > 200 ? 870 : 871))).count != -1) {
         var1.drawRegion(var2.img, 0, this.typePaint * BCBoardScr.c, BCBoardScr.b, BCBoardScr.c, 0, this.x, this.y, 3);
         FontX var3 = Canvas.O;
         if (Canvas.w <= 200) {
            var3 = Canvas.smallFontYellow;
         }

         if (Canvas.stypeInt > 0) {
            var3 = Canvas.normalFont;
         }

         var3.drawString(var1, String.valueOf(this.valuea), this.x, this.y - AvMain.hNormal / 2, 2);
      }

   }
}

File: client/unity/Assets/Scripts/Assembly-CSharp/MoneyPut.cs
public class MoneyPut
{
	public int x;

	public int y;

	public int valuea;

	public int typePaint;

	public static int fl;

	public MoneyPut(int x, int y, int valuea, int typePaint)
	{
		this.x = x;
		this.y = y;
		this.valuea = valuea;
		this.typePaint = typePaint;
	}

	public void paint(MyGraphics g)
	{
		ImageIcon imgIcon = AvatarData.getImgIcon((short)((Canvas.w <= 200) ? 871 : 870));
		if (imgIcon.count != -1)
		{
			g.drawRegion(imgIcon.img, 0f, typePaint * BCBoardScr.hHT, BCBoardScr.rWT, BCBoardScr.hHT, 0, x, y, 3);
			Canvas.smallFontYellow.drawString(g, valuea + string.Empty, x, y - AvMain.hSmall / 2, 2);
		}
	}
}
